test/passing/tests/polytypes-default.ml.ref

```diff
@@ -1,25 +1,25 @@
 let t1 : 'a 'b. 'a t -> b t = ()
 
 let t2 :
-      'a 'b.
-      'a t________________________________ ->
-      'b t_______________________________________ =
+    'a 'b.
```

Comment:
The formatting between locally abstract types and `Ptyp_poly` is different:
```
let foo : type a b c.
    a -> b -> c foooooooo
    =
  ()

let foo :
    'a 'b 'c.
    a -> b -> c foooooooo
    =
  ()
```

I don't know whether this is important or not. The former is really part of the let-binding while the later is part of the type expression.

Reply:
I forgot how the whole thing was designed, one day let's make the time to rewrite these in the parser directly so we can remove the abomination code in Sugar.ml

Reply:
Do you agree in merging it as-is? we can still improve the other case in a separate PR, or better redesign the types later.